extensions/tmtv/src/utils/measurementServiceMappings/measurementServiceMappingsFactory.js

```diff
@@ -0,0 +1,48 @@
+import RectangleROIStartEndThreshold from './RectangleROIStartEndThreshold';
+
+const measurementServiceMappingsFactory = (
+  MeasurementService,
+  DisplaySetService,
+  Cornerstone3DViewportService
+) => {
+  /**
+   * Maps measurement service format object to cornerstone annotation object.
+   *
+   * @param {Measurement} measurement The measurement instance
+   * @param {string} definition The source definition
+   * @return {Object} Cornerstone annotation data
+   */
+
+  const _getValueTypeFromToolType = toolType => {
+    const { SUV_PEAK } = MeasurementService.VALUE_TYPES;
+
+    // TODO -> I get why this was attemped, but its not nearly flexible enough.
+    // A single measurement may have an ellipse + a bidirectional measurement, for instances.
+    // You can't define a bidirectional tool as a single type..
+    const TOOL_TYPE_TO_VALUE_TYPE = {
+      ptSUVPeak: SUV_PEAK,
+    };
+
+    return TOOL_TYPE_TO_VALUE_TYPE[toolType];
```

Comment:
What exactly is this function supposed to be doing? Right now it returns SUV_PEAK if the toolType is ptSUVPeak, so why all the fluff?